unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs

```diff
@@ -0,0 +1,25 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.SettingsPanelHUD.Controls
+{
+    public class SliderSettingsControlView : SettingsControlView
+    {
+        [SerializeField] private Slider slider;
+        [SerializeField] private TextMeshProUGUI indicatorLabel;
+
+        public override void Initialize(SettingsControlModel controlConfig, SettingsControlController settingsControlController)
+        {
+            base.Initialize(controlConfig, settingsControlController);
+
+            slider.onValueChanged.AddListener(sliderValue =>
+            {
+                settingsControlController.OnControlChanged(sliderValue);
+                indicatorLabel.text = sliderValue.ToString("0.0");
```

Comment:
The new mouse sensitivity slider goes from 1 to 100 with whole numbers. I think this should be configurable.

Reply:
Yes! In fact, we can configure it from the specific control configuration asset:
![image](https://user-images.githubusercontent.com/64659061/101761850-f1dde780-3adc-11eb-8313-399fc384d1c4.png)